src/utils/webrtc/simplewebrtc/peer.js

```diff
@@ -87,11 +87,20 @@ function Peer(options) {
 
 util.inherits(Peer, WildEmitter)
 
-function preferH264VideoCodecIfAvailable(offer) {
-	const sdpInfo = sdpTransform.parse(offer.sdp)
+function shouldPreferH264() {
+	try {
+		return initialState.loadState('talk', 'prefer_h264')
+	} catch (exception) {
+		// If the state can not be loaded an exception is thrown
```

Comment:
```suggestion
		// If the state can not be loaded an exception is thrown
		console.debug('Could not find initalState for H.264 preference')
```

let's log this and in the future fix all pages to set this correctly

Reply:
I had the log but forgot to do _git add_... :man_facepalming: